evenodd.c

```diff
@@ -0,0 +1,16 @@
+/*Rebecca Hale, Assignment 2 Part 1*/
```

Comment:
Perfect! Would just suggest adding more description of the program in this comment